### PR TITLE
Issue #501: Set module page columns to a fixed-width.

### LINKED ISSUE #501: [UX] Make sure the columns in the modules page have equal width between categories.
It would be "easier" on our eyes while scrolling this long list...

modules_page-not_equal_widths-narrow
modules_page-not_equal_widths-wide

### PR DESCRIPTION
**Pull request by ghost**
Fixes https://github.com/backdrop/backdrop-issues/issues/501.

### CODE REVIEW

**Comment by klonos** [comment]
Just tested the code by manually applying to an existing installation and it looks great!

...but the css (and thus the widths set in this PR) is only applied to the `td` tags and not the `th` tags. While the `td` tags have meaningful classes, the respective `th` either do not have the same classes or don't have a class at all. It would be easier to target the whole columns in this table if the `th` tags had the same classes as their `td` counterparts (in addition to the "priority" ones they already have).

Hope this makes it easier to understand what I'm saying:

```
<table>
  <thead>
    <tr>
      <th .checkbox><th        ><th          .priority-medium><th              .priority-low><th            >
  <tbody>
    <tr>
      <td .checkbox><th .module><td .version .priority-medium><td .description .priority-low><td .operations>
```

...should be:

```
<table>
  <thead>
    <tr>
      <th .checkbox><th .module><th .version .priority-medium><th .description .priority-low><th .operations>
  <tbody>
    <tr>
      <td .checkbox><th .module><td .version .priority-medium><td .description .priority-low><td .operations>
```

**Comment by klonos** [comment]
...once the classes are in place, the css needs to be changed to:

```
 #system-modules th.checkbox,
 #system-modules td.checkbox {
  width: 20px;
 }
 #system-modules th.module,
 #system-modules td.module {
  width: 180px;
 }
 #system-modules th.version,
 #system-modules td.version {
  width: 70px;
 }
 #system-modules th.operations,
 #system-modules td.operations {
  width: 105px;
}
```

**Comment by klonos** [comment]
...the same meaningful classes need to also be added to the `th` tags of `table .sticky-header`.

File: core/modules/system/css/system.admin.css [hunk 1]
```diff
@@ -90,6 +99,11 @@ span.admin-enabled {
 span.admin-missing {
   color: #f00;
 }
+@media screen and (max-width: 800px) {
+  #system-modules td.module {
+    width: 110px;
+  }
+}
```
Comment (quicksketch) [inline core/modules/system/css/system.admin.css:106]: For consistency with our other CSS, we try to be mobile-first, which means this CSS should be the part outside of the media query, and then put the desktop version in the media query:

``` css
@media screen and (min-width: 800px) {
  #system-modules td.module {
   width: 180px;
  }
}
```

The media-query version should also be right after the non-media-query version of the selector so they can be read together.

Comment (ghost) [inline core/modules/system/css/system.admin.css:106]: @quicksketch Hmm, I was just copying from the other media queries that hide certain columns on smaller screens: https://github.com/backdrop/backdrop/blob/1.x/core/themes/seven/style.css#L476
Might need to create an issue for updating them too.

Comment (quicksketch) [inline core/modules/system/css/system.admin.css:106]: Ah, yeah that's misleading. I agree, we should flip those to be mobile-first as well.

**Comment by quicksketch** [comment]
I flipped the media query and merged this. Thanks @BWPanda!

**Comment by klonos** [comment]
Looks great! Thanx.

**Pull request closed without merging**